docs/development/core/main.toml

```
@@ -0,0 +1,42 @@
+[Database]
+# The type of database to use. Available types: PostgreSql, MySql, SQLite
+type = "PostgreSql"
+# Address to the database
+host = "127.0.0.1"
+# Port of the database
+port = 5438
+# The name of the database
+name = "evosc"
+# Name of the user to access the database
+username = "evosc"
+# Password of the user to access the database
+password = "evosc123!"
+# A string prefix to add to all table names
+tablePrefix = ""
+
+[Logging]
+# Possible values lowest to highest verbosity: none, critical, error, warning, information, debug, trace
+logLevel = "debug"
+# Whether to output logs to the console in JSON.
+useJson = false
+
+[Server]
+# Address to the Trackmania server
+host = "127.0.0.1"
+# Port that the XMLRPC is listening to
+port = 5001
+# Username of the super admin account
+username = "SuperAdmin"
+# Password of the super admin account
+password = "SuperAdmin"
+# If enabled, the client will try to reconnect with the server every 1 second until a connection is established
+retryConnection = true
+
+[Path]
+# Path to the maps folder
+maps = ""
```

```suggestion
maps = "<server root>/UserData/Maps"
```

Thinking we can give a hint at where developers can find the maps directory for servers.

---

Either in the field, or with a comment above. Up to you @dasTor !